openwisp_monitoring/device/static/monitoring/js/chart.css

```
@@ -0,0 +1,156 @@
+}
+.ow-chart .legend span {
+  display: inline-block;
+  width: 10px;
+  height: 10px;
+  margin-right: 2px;
+}
+#ow-chart-contents .circle > span {
+  font-size: 0.15em;
+  word-spacing: -0.75px;
+  color: var(--ow-overlay-60) !important;
+  font-size: 13px;
```

Biome (2.4.7)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)